sleap/gui/commands.py

```diff
@@ -2725,6 +2728,22 @@ def do_action(context, params):
         # Now add the selected instance to the `InstanceGroup`
         context.execute(SetSelectedInstanceGroup, instance_group=instance_group)
 
+#TODO: Add class for changing the name of an instance group
+class SetInstanceGroupName(EditCommand):
```

Comment:
In order to redraw the table after making edits, we need to set the topics list to include UpdateTopic.sessions:
https://github.com/talmolab/sleap/blob/a32d68a1c116584c8950fde841310e77cc71ac99/sleap/gui/app.py#L1288-L1292

```suggestion
class SetInstanceGroupName(EditCommand):
    topics = [UpdateTopic.sessions]
```